comet_core/app.py

```diff
@@ -161,6 +161,7 @@ def __init__(self, database_uri="sqlite://"):
             "owner_reminder_cadence": timedelta(days=7),
             "escalation_time": timedelta(seconds=10),
             "escalation_reminder_cadence": timedelta(days=7),
+            "communication_digest_mode": True,
```

Comment:
I know that the other arguments in `batch_config` is uncommented, but I think it would be easier if we added a comment explaining what `communication_digest_mode` does (similar to the PR description)